core/commands/filestore.go

```diff
@@ -185,6 +186,107 @@ For ERROR entries the error will also be printed to stderr.
 	Type: filestore.ListRes{},
 }
 
+var fixFileStore = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline: "Verify objects in filestore and remove bad links",
+		LongDescription: `
+Verify objects in filestore and remove bad links.
```

Comment:
Let's add a warning about how this may remove pinned data. The user should run `ipfs pin verify` after running this command and correct any issues.